library/ccm.c

```diff
@@ -99,6 +101,11 @@ int mbedtls_ccm_setkey( mbedtls_ccm_context *ctx,
         return( ret );
     }
 
+    if( keybits_dup != keybits || key_dup != key )
+    {
+        return MBEDTLS_ERR_PLATFORM_FAULT_DETECTED;
```

Comment:
@AndrzejKurek , context should be cleaned in all places we detect buffer pointer replacement

Reply:
Please note that there is no such behavior in AES too.

Reply:
@AndrzejKurek ,  I mean that we should clean all data that was copied to our context in all places of PR , including  AES, HMAC, ECC and others

Reply:
@AndrzejKurek , please add this clean up changes to https://github.com/ARMmbed/mbedtls/pull/3481/files